docs/guides/other_libs/efcore.md

```diff
@@ -0,0 +1,112 @@
+---
+uid: Guides.OtherLibs.EFCore
+title: EFCore
+---
+
+# Entity Framework Core
+
+In this guide we will set up EFCore with a PostgreSQL database. Information on other databases will be at the bottom of this page.
+
+## Prerequisites
+
+- A simple bot with dependency injection configured
+- A running PostgreSQL instance
+- [EFCore CLI tools](https://docs.microsoft.com/en-us/ef/core/cli/dotnet#installing-the-tools)
+
+## Downloading the required packages
+
+You can install the following packages through your IDE or go to the nuget link to grab the dotnet cli command.
+
+|Name|Link|
+|--|--|
+| `Microsoft.EntityFrameworkCore` | [link](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore) |
+| `Npgsql.EntityFrameworkCore.PostgreSQL` | [link](https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL)|
+
+## Configuring the DbContext
+
+To use EFCore, you need a DbContext to access everything in your database. The DbContext will look like this. Here is an example entity to show you how you can add more entities yourself later on.
+
+```cs
```

Comment:
Samples should go into /samples and referenced to as `[!code-csharp[title](samples/sample.cs)].`